core/web/middleware.go

```diff
@@ -21,7 +21,7 @@ import (
 // inside this module. To achieve this, we direct webpack to output all of the compiled assets
 // in this module's folder under the "assets" directory.
 
-//go:generate ../../operator_ui/install.sh
+//go:generate make -C ../../ operator-ui
```

Comment:
This seems like an inversion to be calling back to make when we invoke generate from make in the first place. Can we just use `go run` here too?
```suggestion
//go:generate go run ../../ operator-ui/install.go
```

Reply:
I made this inversion to keep all the path resolution happy, i tried to keep it as simple as possible without invoking `git`